packages/wonder-blocks-modal/docs.md

```diff
@@ -37,3 +37,70 @@ class ModalWrapper extends React.Component {
     </ModalLauncher>
 </View>
 ```
+
+### Example: Modals with custom styles 
+
+Sometimes you'll want to customize the styling of the modal .e.g., custom width or height.  You can pass in `style` which will customize the styling of the `ModalDialog`.
```

Comment:
Specifying the component name `ModalDialog` is an implementation detail that probably isn't necessary.

Reply:
updated to modal wrapper